pkg/scheduler/core/generic_scheduler.go

```diff
@@ -564,7 +576,11 @@ func podFitsOnNode(
 			//TODO (yastij) : compute average predicate restrictiveness to export it as Prometheus metric
 			if predicate, exist := predicateFuncs[predicateKey]; exist {
 				if eCacheAvailable {
-					fit, reasons, err = nodeCache.RunPredicate(predicate, predicateKey, pod, metaToUse, nodeInfoToUse, equivClass, cache)
+					predicateGeneration := uint64(0)
```

Comment:
Since zero values are default, the preferred (canonical) way to write this is `var predicateGeneration uint64`.